src/tests/ftest/launch.py

```diff
@@ -29,15 +29,16 @@
 # from util.distro_utils import detect
 # pylint: disable=import-error,no-name-in-module
 from process_core_files import CoreFileProcessing
-from util.logger_utils import get_console_handler, get_file_handler
-from util.results_utils import create_html, create_xml, Job, Results, TestResult
-from util.run_utils import get_local_host, run_local, run_remote, RunException
-from util.user_utils import get_chown_command, get_getent_command, get_groupadd_command, \
-    get_useradd_command, get_userdel_command
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), "util"))
+# pylint: disable=import-outside-toplevel
```

Comment:
I see what you're trying to do here but I don't think that's the correct option, and I'd rather changes to the import/pylint setup were done separately.

See also this line, the util directory should be added to pylint automatically.
https://github.com/daos-stack/daos/blob/master/utils/cq/daos_pylint.py#L369

Reply:
> I don't think that's the correct option

What do you think the correct option is? The only "proper" way I can think of is to convert the entire ftest directory into a python module and install it. The problem is launch.py is one level up from util. And some of the files it imports have other imports from the util directory. And those imports will fail, because the relative import path is one level up.

> I'd rather changes to the import/pylint setup were done separately

Do you mean separate PRs? Wouldn't that introduce a pylint error in the meantime?

> the util directory should be added to pylint automatically

There was somewhere else in the process flagging this, but I don't remember where.

Anyway, these two lines are a partial cherry-pick from https://github.com/daos-stack/daos/pull/10621/files